### PR TITLE
docs: Update log masking policy examples to be Rego v1 compatible

### PR DESCRIPTION
**Pull request by ashutosh-narkar**

### CODE REVIEW

File: docs/content/management-decision-logs.md [hunk 1]
```diff
@@ -224,9 +228,11 @@ the following rule format can be used.
 ```ruby
 package system.log
 
+import rego.v1
+
 # always upsert, no conditions in rule body
-mask[{"op": "upsert", "path": "/input/password", "value": x}] {
-  x := "**REDACTED**"
+mask contains {"op": "upsert", "path": "/input/password", "value": x} if {
```
Comment (anderseknert) [inline docs/content/management-decision-logs.md:234]: While we're at it, I think we could just change this to be:
```rego
mask contains {"op": "upsert", "path": "/input/password", "value": "**REDACTED**"}
```
And do the same at line 218. Not much point in that extra variable.

**Review by anderseknert — DISMISSED** [review]
Thanks! Just a minor suggestion.

File: docs/content/management-decision-logs.md (hunk 1 above)
Comment (ashutosh-narkar) [reply docs/content/management-decision-logs.md]: Updated.

**Review by anderseknert — APPROVED** [review]

File: docs/content/management-decision-logs.md (hunk 1 above)
Comment (anderseknert) [reply docs/content/management-decision-logs.md]: Thanks!

**Pull request merged**